client/src/components/Countdown/Countdown.stories.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+import React from "react";
+
+import { Countdown } from "./Countdown.js";
+
+export default {
+  title: "Countdown",
+  component: Countdown,
+};
+
+// not sure if the component works or not
+const Template = (args) => <Countdown {...args} />;
```

Comment:
Strange, it doesn't work when it first renders inside the storybook, but the countdown starts properly if you hit the reset controls button:
![image](https://user-images.githubusercontent.com/11301919/118045255-abb7d900-b345-11eb-92dc-d3d8f4728cd9.png)
with or without changing the time value.

Reply:
Aha, looks like a bug in `Countdown` actually. At least, pretty sure this isn't intentional (although, unintentionally perhaps, it _does_ work everywhere we use it right now).

Basically, all the logic to initialize and manage the countdown logic is inside its `componentDidUpdate`... so the component doesn't actually start counting down until after the first time it updates. Probably just need to move that logic to a separate method, and then call it from `componentDidMount` in addition to `componentDidUpdate`.

Should investigate the actual use in the site a bit more if you fix that, just to be careful. Something may (un)intentionally rely on the current behaviour.

Reply:
I have fixed the bug, but I have yet to find it on the GC Infobase website.